packages/react-emotion/test/__snapshots__/index.test.js.snap

```diff
@@ -373,14 +373,18 @@ exports[`styled withComponent will replace tags but keep styling classes 1`] = `
   color: green;
 }
 
+.emotion-2 {
+  color: green;
+}
```

Comment:
If anyone's curious why this snapshot changed and two class names with the same styles exist is that they each have different labels and the optimization that was here broke the label but now that it's not here, the label works so there are two different classes.

Reply:
is it possible to add a test confirming the correct behaviour in previously broken case? except having a different class name here now this test doesnt actually change the observable output